gce/notebook/Dockerfile

```diff
@@ -8,12 +8,12 @@ RUN apt-get update \
 USER $NB_USER
 
 RUN conda install --yes -c ioam -c bokeh/channel/dev \
-    bokeh=0.12.15dev1 \
+    bokeh=0.12.15dev3 \
     cytoolz \
     datashader \
     dask=0.17.2 \
     dask-ml \
-    distributed=1.21.4 \
+    distributed=1.21.6 \
```

Comment:
@mrocklin - can you confirm that this version of distributed should work with `bokeh=0.12.15dev3`